src/workScheduleDayDefinitions/index.ts

```diff
@@ -10,12 +9,19 @@ type ResourceName = typeof resourceName;
 export class WorkScheduleDayDefinitionsEndpoint extends BaseApi<ResourceName> {
   public readonly resourceName = resourceName;
 
-  public create(
-    data: WorkScheduleDayDefinitionCreate,
-    params?: RequestParams<Entity<ResourceName>>,
-  ): Required<ResourceName, Entity<ResourceName>[]> {
-    const response = this._post<ResourceName>('create', data, params);
-    return required(response);
+  public async create(
+    data: WorkScheduleDayDefinitionCreate | WorkScheduleDayDefinitionCreate[],
+  ): Promise<
+    | LibraryReturn<'workScheduleDayDefinitions', WorkScheduleDayDefinition>
+    | LibraryReturn<'workScheduleDayDefinitions', (ParsedErrorMesage | WorkScheduleDayDefinition)[]>
+  > {
+    if (Array.isArray(data)) {
+      const response = this._postBatch<ResourceName>('create', data);
+      return requiredBatch(response);
+    } else {
+      const response = this._post<ResourceName>('create', data);
+      return requiredSingle(response);
+    }
```

Comment:
Should we maybe create a separate `createBatch` endpoint? Because for now you will need to add more `if` for the method responses and so on

Reply:
Since we have the same approach for Update, I merged them together, but in general it is a good idea